pom.xml

```diff
@@ -35,7 +36,7 @@
         <structured-logging.version>3.0.1</structured-logging.version>
         <api-security-java.version>2.0.1</api-security-java.version>
         <api-helper-java.version>3.0.0</api-helper-java.version>
-        <api-sdk-java.version>6.0.2</api-sdk-java.version>
+        <api-sdk-java.version>unversioned</api-sdk-java.version>
```

Comment:
isn't this 6.0.6 now? or a 6.0.7 has just gone in with some other stuff.

Reply:
Yes it should be 6.0.6, i'll update that.